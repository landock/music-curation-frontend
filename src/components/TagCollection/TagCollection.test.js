import React from 'react';
import { render } from 'react-testing-library';
import renderer from 'react-test-renderer';
// import TagCollection from '.';
it('TagCollection: default', () => {
  let tagCollectionProps = {
    tags: [{ id: 1, title: 'Mock' }, { id: 2, title: 'Test' }],
  };
  // const component = renderer.create(<TagCollection {...tagCollectionProps} />);
  // const tree = component.toJSON();
  // expect(tree).toMatchSnapshot();
});
