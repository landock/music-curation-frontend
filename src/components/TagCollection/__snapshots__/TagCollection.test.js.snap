// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TagCollection: default 1`] = `
Array [
  <a
    className="ui red label"
    onClick={[Function]}
    style={
        Object {
            "marginBottom": "1%",
          }
    }
>
    Gorgeous Fresh Pizza
</a>,
  <a
    className="ui red label"
    onClick={[Function]}
    style={
        Object {
            "marginBottom": "1%",
          }
    }
>
    Cambridgeshire
</a>,
  <a
    className="ui red label"
    onClick={[Function]}
    style={
        Object {
            "marginBottom": "1%",
          }
    }
>
    yellow
</a>,
  <a
    className="ui red label"
    onClick={[Function]}
    style={
        Object {
            "marginBottom": "1%",
          }
    }
>
    sky blue
</a>,
]
`;
